APIs/SentinelHub/Byoc/Examples.qmd

```diff
@@ -1,622 +1,182 @@
-
-## Converting to COG
-
-### Constraints and settings
-
-COGs can contain either a single band or multiple bands. For multi-band
-COGs we support both [planar configurations
-formats](https://www.awaresystems.be/imaging/tiff/tifftags/planarconfiguration.html){target="_blank"} -
-chunky and planar format.
-
-There are a few additional constraints in addition to having COG files.
-These are:
-
--   The COG header size must not exceed one megabyte.
--   The internal tile size must be between 256 x 256 and 2048 x 2048.
--   The projection needs to be one of: WGS84 (EPSG:4326), WebMercator
-    (EPGS:3857), any UTM zone (EPSG:32601-32660, 32701-32760), or Europe
-    LAEA (EPSG:3035).
--   The COG must not cross any of the two poles.
--   The band name should be a valid JavaScript identifier so it can be
-    safely used in evalscripts; valid identifiers are case-sensitive,
-    can contain Unicode letters, \$, \_, and digits (0-9), but may not
-    start with a digit, and should not be one of the reserved JavaScript
-    keywords.
--   There can be at most 100 bands.
--   Multi-band COGs in chunky format can have at most 10 bands.
--   The file names need to be consistent for all tiles in a collection.
-    For example, if you have B1.tiff in one tile then you also need
-    B1.tiff in all the other tiles in your collection.
--   All files of each tile needs to have consistent extension (so a tile
-    cannot contain both B1.tiff and B2.TIF).
--   The maximum allowable difference between the intersection of all
-    file bounding boxes and each individual file is one pixel of that
-    file \[1\].
--   All files of each band need to have the same bit depth.
--   Files can be compressed with DEFLATE, ZLIB, PIXTIFF_ZIP, PACKBITS or
-    LZW compression method. JPEG compression is not supported.
--   Supported sample types and bit depths are the same as those
-    supported for outputs, as well as reading unsigned integer 1, 2, 4
-    bit files. See also
-    [sampleType](/APIs/SentinelHub/Evalscript/V3.qmd#sampletype).
-
-Bands can have different resolutions.
-
-For best performance we recommend the following setting for COGs:
-deflate compressed with 1024x1024 pixel internal tiling.
-
--   \[1\]: Here\'s one example of files with slightly different bounding
-    boxes. One file has the bounding box `[0, 0, 10, 10]` and resolution
-    of one meter per pixel, and the other file has the bounding box
-    `[0.5, 0.5, 10.5, 10.5]` and 0.5 meter resolution. The intersection
-    `[0.5, 0.5, 10, 10]` is not more than one pixel away from each
-    individual file, therefore such files are valid for BYOC.
-
-### GDAL example command
-
-COGs can be generated in a single step with GDAL 3.1 or newer using the
-[COG raster
-driver](https://gdal.org/drivers/raster/cog.html){target="_blank"}. For
-older GDAL versions or if you want planar multi-band COGs, see
-[below](#older-gdal-versions-or-planar-multi-band-cogs). **Even though
```

Comment:
How about

::: {.callout-tips}

Even though you can use any GDAL version, we highly recommend you use v3.1 or newer, as older versions have issues with average downsampling (see https://gdal.org/programs/gdaladdo.html

:::